web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx

```diff
@@ -48,12 +48,13 @@ export default function Advanced(props: AdvancedProps) {
 
   const [error, setError] = useState<InputsErrors | null>(null);
   const [fees, setFees] = useState<string>(initialFees);
-  const [feesField, setFeesField] = useState<string>('');
-  const [customFees, setCustomFees] = useState<boolean>(initialCustomFees);
+  const [customFees, setAdvancedFees] = useState<string>('');
```

Comment:
the good practice would be customFees, setCustoFees or advancedFees, setAdvancedFees